content/docs/user-guide/project-config/cli-reference.md

```diff
@@ -2,6 +2,1344 @@
+
+- Launch python and run the `o3de.py` script.
+
+    ```cmd
+    <engine>/python/python.cmd <engine>/scripts/o3de.py <command>
+    ```
+
+- Run the `o3de.bat` batch file directly.
+
+    ```cmd
+    <engine>/scripts/o3de.bat <command>
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```cmd
+<engine>/scripts/o3de.bat register --this-engine
+```
+
+{{% /tab %}}
+{{% tab name="Linux" %}}
+
+For Linux, open a terminal. Then, run the O3DE Python runtime using one of the following ways, and enter the command you want to perform. Note: Replace `<engine>` with the path to your engine. 
+
+- Launch python and run the `o3de.py` script.
+
+    ```bash
+    <engine>/python/python.sh <engine/scripts/o3de.py
+    ```
+
+- Run the `o3de.sh` shell script file directly.
+
+    ```bash
+    <engine>/scripts/o3de.sh
+    ```
+
+**Example**
+
+For example, to register the engine, you can enter the following command. 
+
+```bash
+<engine>/scripts/o3de.sh register --this-engine
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+## Commands
+
+
+| Command | Description | 
+| - | - |
+| [`get-global-project`](#get-global-project) | Gets the global project that is registered to the engine. |
+| [`set-global-project`](#set-global-project) | Sets the specified project as the engine's global project. |
+| [`create-template`](#create-template) | Creates a template out of the specified source path. |
+| [`create-from-template`](#create-from-template) | Creates a template from an existing template.  |
+| [`create-project`](#create-project) | Creates a new project. |
+| [`create-gem`](#create-gem) | Creates a new Gem. |
+| [`register`](#register) | Register an O3DE object. |
+| [`register-show`](#register-show) | Shows the registered O3DE objects. |
+| [`get-registered`](#get-registered) | Shows the registered O3DE object's path that is mapped to the specified name. |
+| [`enable-gem`](#enable-gem) | Enables the Gem in your project, so you can use the assets or code the Gem provides. |
+| [`disable-gem`](#disable-gem) | Disables the Gem in your project. |
+| [`sha256`](#sha256) | Creates a secure hash algorithm (SHA-256). |
+<!-- | [`edit-project-properties`](#edit-project-properties) |  | -->
+
+
+
+## `get-global-project`
+
+Gets the global project that is registered to the engine in the specified `.setreg` file at `<user>/.o3de/Registry/`.
+
+
+```cmd
+usage: o3de.py get-global-project [-h] [-i INPUT_PATH]
+```
+
+### Usage
+
+```cmd
+o3de.bat get-global-project -i SETREG_PATH
+```
+
+### Optional Parameters
+
+`-h, --help`
+show this help message and exit
+
+`-i INPUT_PATH, --input-path INPUT_PATH`
+
+Optional path to file to read `/Amazon/AzCore/Bootstrap/project_path` key from. If not supplied, then
+`C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg` is used instead
+
+
+
+## `set-global-project`
+
+Sets the specified project as the engine's global project in the specified `.setreg` file at `<user>/.o3de/Registry/`
```

Comment:
```suggestion
Sets the specified project as the engine's global project. By default this is set in `C:\Users\USER_PATH\.o3de\Registry\bootstrap.setreg`. You can also specify a file path using `-o`.
```

Reply:
This is only the path for setting the engine's global project on Windows.
On Mac and Linux the paths are based on their home directory structure
Linux: `/home/USER_PATH/.o3de/Registry/bootstrap.setreg`
Mac: `/Users/USER_PATH/.o3de/Registry/bootstrap.setreg`

Reply:
@lumberyard-employee-dm brings up a good point. Although our Windows support is currently more established than Linux and Mac, I think our docs should be platform inclusive (when possible). In some cases, I think it's okay to just show the Windows version, as long as customers have already been given Mac and Linux alternatives. For example, the usage examples use `o3de.bat`, which is Windows-based. However, in the beginning of this document, we mention how to run `o3de` on Linux platforms. So, we already gave customers who use Linux an alternative instruction. 

One solution I can think of is writing the path as platform independent. For example, `<USER_PATH>\.o3de\Registry\bootstrap.setreg`. Is this format understandable across the different platforms?